app/apollo/models/user.local.schema.js

```diff
@@ -298,10 +327,20 @@ UserLocalSchema.statics.isAuthorizedBatch = async function(me, orgId, objectArra
   });
 
   if (orgMeta) {
+    const userAuthorizationArray = me.meta.orgs.find(org => org.authorization);
```

Comment:
This seems wrong -- the variable isnt an Array, and the way `find()` is used isn't correct (and/or it's non-obvious what the code attempting).  This line and section needs revision and/or comments.

Reply:
Still needs revision (currently changed to `const userAuthObject = (me.meta.orgs.length > 0 ? me.meta.orgs[0] : null);`).
- If it's getting an _org_ from a list of orgs, the variable name shouldn't call it an "auth object" -- just call it `const org`.
- The way the org is retrieved is still incorrect -- always using the first org when there could be multiple would incorrectly apply a user's auth from the first org when querying items from a different one.
- There's apparently no need to find the org again -- it was already found earlier in the function.

Reply:
user.local.schema.js is now using the previously found org.

Reply:
resolved